datasource/remote/src/main/java/com/egdroid/datasource/remote/utils/NetworkUtils.kt

```diff
@@ -0,0 +1,15 @@
+package com.egdroid.datasource.remote.utils
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+
+object NetworkUtils {
```

Comment:
Try to make something like this as an extension if you are going to use Kotlin 😄